src/scss/components/_listings.scss

```diff
@@ -50,21 +80,149 @@ $listing-borderColor: $color-border;
             }
         }
 
+        .c-listing-item-img {
+            float: left;
+            width: 50px;
+
+            @include media('>wide') {
+                width: 65px;
+            }
+        }
+
+        .c-listing-item-imgItem {
+            width: 100%;
+            height: auto;
+        }
+
         .c-listing-item-info {
+            top: -4px;
+            position: relative;
             color: $color-text;
-            flex-wrap: nowrap;
-            margin-left: spacing(base);
+            padding-left: 66px;
+
+            @include media('>wide') {
+                position: relative;
+                padding-left: 80px;
+                padding-right: 260px;
+
+                &:after {
+                    content: '';
+                    top: 4px;
+                    width: 1px;
+                    right: 259px;
+                    height: 100%;
+                    display: block;
+                    position: absolute;
+                    background: $listing-borderColor;
+
+                    .c-listing--offline & {
+                        background: $listing-offlineBorderColor;
+                    }
+                }
+            }
+        }
+
+        .c-listing-item-title {
+            word-break: break-word;
         }
+
+        .c-listing-item-meta {
+            top: -3px;
+            float: right;
+            position: relative;
+
+            @include media('>wide') {
+                top: auto;
+                width: 260px;
+                padding-bottom: 50px;
+                padding-left: spacing(x2);
+                border-left: 1px solid $listing-borderColor;
+
+                .c-listing--offline & {
+                    border-color: $listing-offlineBorderColor;
+                }
+            }
+        }
+
+        .c-listing-item-rating {
+            padding: 2px 0 3px;
+
+            .c-rating,
```

Comment:
Should try not to mix component references, as it helps when removing/amending them later.

Hard to see what's being targetted without markup, but is there another way of doing this (using classnames within `c-listing`)